File: Chat/ChatParser/src/FriendMessageStatusDTO.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   FriendMessageStatusDTO.cpp
 * Author: noman
 * 
 * Created on January 25, 2016, 1:14 PM
 */

#include "FriendMessageStatusDTO.h"

namespace imsdk {
FriendMessageStatusDTO::FriendMessageStatusDTO() {
}

FriendMessageStatusDTO::~FriendMessageStatusDTO() {
}

void FriendMessageStatusDTO::setMessageStatus(int messageStatus) {
    this->messageStatus = messageStatus;
}

int FriendMessageStatusDTO::getMessageStatus() const{
    return messageStatus;
}

void FriendMessageStatusDTO::setUpdateDate(long long updateDate) {
    this->updateDate = updateDate;
}

long long FriendMessageStatusDTO::getUpdateDate() const{
    return updateDate;
}

void FriendMessageStatusDTO::setPacketId(std::string packetId) {
    this->packetId = packetId;
}

std::string FriendMessageStatusDTO::getPacketId() const{
    return packetId;
}
}

File: Chat/Chat/wrapper/java/JChatSDK/src/com/ringid/imsdk/FriendMessageStatusDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.ringid.imsdk;

/**
 *
 * @author noman
 */
public class FriendMessageStatusDTO {
    
    private String packetId;
    private long updateDate;
    private int messageStatus;
    
    public FriendMessageStatusDTO(){
        
    }
    
    public FriendMessageStatusDTO(String packetId, long updateDate, int messageStatus){
        this.packetId = packetId;
        this.updateDate = updateDate;
        this.messageStatus = messageStatus;
    }

    /**
     * @return the packetId
     */
    public String getPacketId() {
        return packetId;
    }

    /**
     * @param packetId the packetId to set
     */
    public void setPacketId(String packetId) {
        this.packetId = packetId;
    }

    /**
     * @return the updateDate
     */
    public long getUpdateDate() {
        return updateDate;
    }

    /**
     * @param updateDate the updateDate to set
     */
    public void setUpdateDate(long updateDate) {
        this.updateDate = updateDate;
    }

    /**
     * @return the messageStatus
     */
    public int getMessageStatus() {
        return messageStatus;
    }

    /**
     * @param messageStatus the messageStatus to set
     */
    public void setMessageStatus(int messageStatus) {
        this.messageStatus = messageStatus;
    }
    
}
